custom/icds_reports/static/js/icds_app.js

```diff
@@ -10,6 +10,16 @@ function MainController($scope, $route, $routeParams, $location, $uibModal, $win
     $scope.healthCollapsed = true;
     $scope.isWebUser = isWebUser;
 
+    var locationParams = $location.search();
+    var newKey;
+    for (var key in locationParams) {
+        newKey = key;
+        if (newKey.startsWith('amp;')) {
```

Comment:
Why are these `amp;` showing up? Are we generating links that don't properly escape `&` ?

Reply:
Actually, I have no idea. Sorry